app/views/content_items/how_government_works.html.erb

```diff
@@ -0,0 +1,552 @@
+
+        <%= render "govuk_publishing_components/components/list", {
+          extra_spacing: true,
+          visible_counters: true,
+          items: [
+            "health",
+            "education",
+            "culture",
+            "the environment",
+            "transport"
+          ]
+        } %>
+      </section>
+
+      <section class="govuk-!-margin-bottom-9">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Local government",
+          id: "local-government",
+          font_size: "l",
+          margin_bottom: 4,
+        } %>
+
+        <p class="govuk-body">Councils make and carry out decisions on local services. Many parts of England have 2 tiers of local government: county councils and district, borough or city councils.</p>
+        <p class="govuk-body">In some parts of the country, there’s just one tier of local government providing all the functions, known as a ‘unitary authority’. This can be a city, borough or county council – or it may just be called ‘council’. As well as these, many areas also have parish or town councils.</p>
+        <p class="govuk-body"><%= link_to "Understand how your council works", "/understand-how-your-council-works/types-of-council/", class: "govuk-link" %></p>
+      </section>
+
+      <section class="govuk-!-margin-bottom-9">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Parliament",
+          id: "parliament",
+          font_size: "l",
+          margin_bottom: 4,
+        } %>
+
+        <p class="govuk-body">Parliament is separate from government. Made up of the House of Commons and the House of Lords, its role is to:</p>
+
+        <%= render "govuk_publishing_components/components/list", {
+          extra_spacing: true,
+          visible_counters: true,
+          items: [
+            "look at what the government is doing",
+            "debate issues and pass new laws",
+            "set taxes"
+          ]
+        } %>
+
+        <p class="govuk-body">
+          <%= link_to "Find out about how Parliament works", "https://www.parliament.uk/", class: "govuk-link", rel: "external" %>
+        </p>
+
+        <p class="govuk-body">Read The Official Report at <%= link_to "Hansard", "http://www.parliament.uk/business/publications/", class: "govuk-link", rel: "external" %></p>
+      </section>
+
+      <section>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "History of government",
+          id: "history-uk-government",
+          font_size: "l",
+          margin_bottom: 4,
+        } %>
+
+        <p class="govuk-body">Britain has one of the oldest governments in the world. Find out more about how it has worked and who has shaped it in the <%= link_to "history", "/government/history/", class: "govuk-link" %> section.</p>
```

Comment:
the href `/government/history` has a rogue `/` at the end of it. I know it still works, but whilst you're in the area!

Reply:
Cool, I'll remove this (more copy-and-paste from Whitehall inconsistencies!)